plugins/com.google.cloud.tools.eclipse.appengine.newproject.test/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProjectTest.java

```diff
@@ -12,6 +22,27 @@
 
   @Mock private IAdaptable adaptable;
 
+  private SubMonitor monitor = SubMonitor.convert(new NullProgressMonitor());
+  private IProject project;
+  
+  @Before
+  public void setUp() throws CoreException {
+    IWorkspace workspace = ResourcesPlugin.getWorkspace();
+    project = workspace.getRoot().getProject("foobar");
+  }
+  
+  @After
+  public void cleanUp() throws CoreException {
+    project.delete(true, monitor);
+  }
+  
+  @Test
+  public void testConstructor() throws InvocationTargetException, CoreException {
+    AppEngineStandardProjectConfig config = new AppEngineStandardProjectConfig();
+    config.setProject(project);
+    new CreateAppEngineStandardWtpProject(config, adaptable);
```

Comment:
Do you have the trace?

> On May 6, 2016, at 3:32 AM, Elliotte Rusty Harold <notifications@github.com> wrote:
> 
> In plugins/com.google.cloud.tools.eclipse.appengine.newproject.test/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProjectTest.java:
> 
> > +  @Before
> > +  public void setUp() throws CoreException {
> > +    IWorkspace workspace = ResourcesPlugin.getWorkspace();
> > +    project = workspace.getRoot().getProject("foobar");
> > +  }
> > +  
> > +  @After
> > +  public void cleanUp() throws CoreException {
> > +    project.delete(true, monitor);
> > +  }
> > +  
> > +  @Test
> > +  public void testConstructor() throws InvocationTargetException, CoreException {
> > +    AppEngineStandardProjectConfig config = new AppEngineStandardProjectConfig();
> > +    config.setProject(project);
> > +    new CreateAppEngineStandardWtpProject(config, adaptable);
> I did add a test like that. It passed in Eclipse. However it failed with a NullPointerException deep inside the Eclipse classes when run from maven. Haven't figured that out yet.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>